operations/mimir/read-write-deployment/backend.libsonnet

```diff
@@ -35,9 +35,9 @@
       'query-scheduler.max-used-instances': 2,
     },
 
-  mimir_backend_zone_a_args:: {},
-  mimir_backend_zone_b_args:: {},
-  mimir_backend_zone_c_args:: {},
+  mimir_backend_zone_a_args:: $.store_gateway_zone_a_args {},
+  mimir_backend_zone_b_args:: $.store_gateway_zone_b_args {},
+  mimir_backend_zone_c_args:: $.store_gateway_zone_c_args {},
```

Comment:
Should this also include `compactor_args`, `ruler_args`, `query_scheduler_args` too?

similar comment for `mimir_write_zone_*_args` and `distributor_args`

Reply:
> Should this also include compactor_args, ruler_args, query_scheduler_args too?

It's already done, when we initialise the `mimir_backend_args` and `mimir_write_args` (see few lines above). We were not merging the per-zone flags, which is what this PR fixes.

Reply:
oh, i see 👍